src/surfaces/actionBar/actionBarSearch.jsx

```diff
@@ -24,6 +25,7 @@ const defaultProps = {
     isMobileSearch: false,
     isMobileSearchVisible: false,
     onClearClick: undefined,
+    onFocused: undefined,
```

Comment:
So do we deliberately want past tense here?

Reply:
No, we usually use present tense for this kinds of things, right?  I think request a change; should be easy enough.

Reply:
renamed

Reply:
Muchas gracias! =)